pyoptsparse/pySNOPT/pySNOPT.py

```diff
@@ -644,33 +646,6 @@ def _set_snopt_options(self, iPrint, iSumm, cw, iw, rw):
             elif isinstance(value, type(None)):
                 snopt.snset(name, iPrint, iSumm, inform, cw, iw, rw)
 
-        return
-
-    def _on_getInform(self, infocode):
```

Comment:
This code was never used, and I figured we can probably just use the super() implementation.